LibSWBF2/CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ set(FMT_DIR "${PROJECT_SOURCE_DIR}/../ThirdParty/fmt")
 add_subdirectory(${GLM_DIR} "${CMAKE_CURRENT_BINARY_DIR}/glm")
 add_subdirectory(${FMT_DIR} "${CMAKE_CURRENT_BINARY_DIR}/fmt")
 
+
+target_compile_definitions(LibSWBF2 PRIVATE LOOKUP_CSV_PATH="${PROJECT_SOURCE_DIR}/../lookup.csv")
+
+
```

Comment:
Needed because FNV lookup table initialization requires absolute path (at least on my system or because the cmake root is a dir lower now)